script/binary

```diff
@@ -17,7 +17,7 @@
 set -e
 
 
-GIT_COMMIT=$(git describe --tags --dirty)
+GIT_COMMIT=$(git rev-parse --short HEAD)
```

Comment:
fyi the old git command simply doesn't seem to work on github action builders, not entirely sure why

fails saying nothing to describe